.github/actions/templates/getWorkflowInput/scripts/Get-WorkflowDefaultInput.ps1

```diff
@@ -1,3 +1,46 @@
+<#
+.SYNOPSIS
+Retrieve default value for a specified input in a workflow.
+
+.PARAMETER InputName
+Mandatory. The name of the input to get the default value for.
+
+.PARAMETER Content
+Mandatory. The content of the GitHub workflow file.
+
+.PARAMETER Lines
+Mandatory. The number of lines to check after finding the specified input.
+
+.EXAMPLE
+$content = Get-Content -Path .\workflow.yml
+Get-DefaultValue -Text 'removeDeployment' -Content $Content
+
+Retrieve input default values for the 'removeDeployment' in the workflow.yml file.
+#>
+function Get-DefaultValue {
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory)]
+        [string] $InputName,
+        [Parameter(Mandatory)]
+        [string[]] $Content,
+        [Parameter()]
+        [int] $Lines = 3
```

Comment:
this will work for now - just wonder if it is worth thinking about the possiblity that one might now define a default for a parameter/description or something else.
Should we instead do some sort of while loop? E.g. `while ('element[i] -notlike "default:*" { $index++} )'`.
Thoughts?

Reply:
let's improve on that some other time

Reply:
Wont fix now. Make an issue :D